  # path point
   PathPoint path_point

  # linear velocity
   float64 v  # in [m/s]
  # linear acceleration
   float64 a
  # relative time from beginning of the trajectory
   float64 relative_time